docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -88,21 +56,21 @@ _(these items were already deprecated and did nothing)_
 - `<option name="giveUnitControl" value="true"/>`
 - `<option name="giveUnitControl" value="false"/>`
 
-### Changes of map (properties) file items:
+## map.properties Changes
 
 Within every "map" properties file, inside the main folder of the map, change:
 
 - `color.Impassible=` >> `color.Impassable=`
 
-### Changes of sounds (properties) file items:
```

Comment:
This whole section has been removed without a valid substutite, so it is now a missing item, thus needs to be added back. To clarify, what this section was doing, was telling that if having a code like
`game_start=classical/game_start/game_start_02_gladiator_entrance.wav`
within the "sounds" properties file, the code has to be changed to
`game_start=classical/game_start/game_start_02_gladiator_entrance.mp3`.
This is not even necessarily related to the "sounds" folder, as, in the example, we are referencing code in the TripleA "assets" folder.

Reply:
The existing was not sufficient that because you can't just change the file extension. There is now a mention that you need to re-encode the wav files. Mentioning that the files are referenced in the sound properties file would probably be good.

Reply:
The existing section was only telling to open the "sounds" properties file and change any occurrence of ".wav" to ".mp3". Then there was another section telling to change all files within the "sounds" folder from "wav" to "mp3". I'm not seeing how those two sections were not fully sufficient (and they are independent, as you can have a map with "wav" references in the "sounds" properties file but no "sounds" folder (the example I made is "270BC", in High Quality) as well as a map with "sounds" in folder and no specific references to any of them in the "sounds" properties file (you can reference whole folders and subfolders, instead of specific sound files, and that would not need any upgrade, as far as I know)).